ts/features/bonus/bpd/screens/onboarding/BpdCTAStartOnboardingScreen.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import { Dispatch } from "redux";
+import { connect } from "react-redux";
+import * as pot from "italia-ts-commons/lib/pot";
+import { bpdOnboardingStart } from "../../store/actions/onboarding";
+import { LoadingErrorComponent } from "../../../bonusVacanze/components/loadingErrorScreen/LoadingErrorComponent";
+import I18n from "../../../../../i18n";
+import { loadAvailableBonuses } from "../../../bonusVacanze/store/actions/bonusVacanze";
+import { GlobalState } from "../../../../../store/reducers/types";
+import { availableBonusTypesSelector } from "../../../bonusVacanze/store/reducers/availableBonusesTypes";
+import { useActionOnFocus } from "../../../../../utils/hooks/useOnFocus";
+import { isStrictSome } from "../../../../../utils/pot";
+
+export type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+const loadingCaption = () => I18n.t("global.remoteStates.loading");
+
+/**
+ * this is a dummy screen reachable only from a message CTA
+ */
+const BpdCTAStartOnboardingScreen: React.FC<Props> = (props: Props) => {
+  const hasError = () => pot.isError(props.availableBonus);
+
+  // load available bonus when component is focused
+  useActionOnFocus(props.loadAvailableBonus);
+
+  React.useEffect(() => {
+    // bpdOnboardingStart navigate to ToS screen that needs availableBonus data
+    if (isStrictSome(props.availableBonus)) {
+      props.startBpd();
+    }
+  }, [props.availableBonus]);
+
+  return (
+    <LoadingErrorComponent
```

Comment:
The feature added by this PR works awesome!

Anyway if something goes bad, user would be stuck in this screen with only the retry button available, may we wrap this component with a `BaseScreenComponent` so that we may have the back button on top, what do you think?

<img src="https://user-images.githubusercontent.com/3959405/100780904-9dff3e80-340a-11eb-9b6c-ec9d18f35f73.png" height="550"/>

Reply:
Nice catch!
I'll add it in the next commit

Reply:
<img src="https://user-images.githubusercontent.com/822471/100782367-98a2f380-340c-11eb-99cf-e5809f321db7.gif" height="600"/>